modules/combined/tests/frictional_contact/single_point_2d/tests

```diff
@@ -4,13 +4,28 @@
     input = 'single_point_2d.i'
     exodiff = 'single_point_2d_out.e'
     custom_cmp = 'single_point_2d.cmp'
-    max_parallel = 1
+    superlu = true
+  [../]
+  [./fcp]
+    type = 'Exodiff'
+    input = 'single_point_2d_fcp.i'
+    exodiff = 'single_point_2d_fcp_out.e'
+    custom_cmp = 'single_point_2d.cmp'
+    superlu = true
   [../]
   [./predictor]
     type = 'Exodiff'
     input = 'single_point_2d_predictor.i'
     exodiff = 'single_point_2d_predictor_out.e'
     custom_cmp = 'single_point_2d.cmp'
+    superlu = true
+  [../]
+  [./fcp_predictor]
+    type = 'Exodiff'
+    input = 'single_point_2d_fcp_predictor.i'
+    exodiff = 'single_point_2d_fcp_predictor_out.e'
+    custom_cmp = 'single_point_2d.cmp'
+    superlu = true
     max_parallel = 1
```

Comment:
We actually do need that because there's a parallel bug in that code (which is the old method I'm replacing with this new work). I'll make an issue for that bug.